config/sources/families/rockchip-rk3588.conf

```diff
@@ -41,20 +41,27 @@ case $BRANCH in
 
 	edge)
 		LINUXFAMILY=rockchip-rk3588
-		LINUXCONFIG='linux-rockchip-rk3588-'$BRANCH
-		KERNEL_MAJOR_MINOR="6.8" # Major and minor versions of this kernel.
-		KERNELPATCHDIR='rockchip-rk3588-edge'
+		KERNEL_MAJOR_MINOR="6.10"                                 # Major and minor versions of this kernel.
+		LINUXCONFIG="linux-rockchip-rk3588-${KERNEL_MAJOR_MINOR}" # Attention: not -${BRANCH} (edge/current/legacy), but -${KERNEL_MAJOR_MINOR} thus 6.8 / 6.10 etc
+		# No need, because default: KERNELPATCHDIR='archive/rockchip-rk3588-6.10'
```

Comment:
> A folder called `archive` implicates something older, not actively worked on. I believe we should re-think this approach.

Discussed in the past. _All other kernels_ are in `archive/` right now if you notice. One day we rename `archive` to `per-version` for naming clarity, but otherwise, that's what it already is.

Reply:
For completeness: still living outside of `archive/`:

- media-current / media-edge / media-legacy (long-abandoned tree, hopefully soon to be removed)
- rk35xx-legacy / rk35xx-vendor-6.1 (one already has even the version in there and could be easily moved to archive)
- rockchip-rk3588-collabora (really just for added-DT's, as we don't wanna patch Collabora ever)
- rockchip-rk3588-edge (dying here)
- s5p6818-legacy ( ahhn... what's this? )

Reply:
Exactly. First, all go into archive/, and then archive/ disappears.